packages/core-data/src/index.js

```diff
@@ -22,23 +22,29 @@ import { unlock } from './private-apis';
 // The "kind" and the "name" of the entity are combined to generate these shortcuts.
 
 const entitySelectors = rootEntitiesConfig.reduce( ( result, entity ) => {
-	const { kind, name } = entity;
+	const { kind, name, plural } = entity;
 	result[ getMethodName( kind, name ) ] = ( state, key, query ) =>
 		selectors.getEntityRecord( state, kind, name, key, query );
-	result[ getMethodName( kind, name, 'get', true ) ] = ( state, query ) =>
-		selectors.getEntityRecords( state, kind, name, query );
+
+	if ( plural ) {
```

Comment:
Why is this check necessary? Don't all root entities have `plural` now? Is it in case we add an entity in the future and it doesn't have a `plural` defined?

Reply:
I guess Riad answered most of my question here - we intentionally don't want plural support for some of the entities.

Reply:
I mentioned this in the description, but I probably should've added an inline comment as well. Not every entity will have "plural" selectors. Current cases are `site` and `__unstableBase`, but we have to maintain them for BC.